File: azure/transporterSolution/transporterService/DataObjects/ProductAddressItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class ProductAddressItem : BaseItem
    {
        public string Address { get; set; }
        public string Number { get; set; }
        public string Location { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string PostalCode { get; set; }
        public string Reference { get; set; }

        public virtual ProductDetailItem ProductDetail { get; set; }

    }
}

File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/ProductAddressItem.java
package com.example.fugliara.ccs002.server.dataobjects;

public class ProductAddressItem {
    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("address")
    public String address;
    @com.google.gson.annotations.SerializedName("number")
    public String number;
    @com.google.gson.annotations.SerializedName("location")
    public String location;
    @com.google.gson.annotations.SerializedName("city")
    public String city;
    @com.google.gson.annotations.SerializedName("state")
    public String state;
    @com.google.gson.annotations.SerializedName("postalCode")
    public String postalCode;
    @com.google.gson.annotations.SerializedName("reference")
    public String reference;
    @com.google.gson.annotations.SerializedName("productDetailItem")
    public ProductDetailItem productDetailItem;

    public ProductAddressItem() {
        id = "";
        address = "";
        number = "";
        location = "";
        city = "";
        state = "";
        postalCode = "";
        reference = "";
        productDetailItem = null;
    }
}
